modules/assim.batch/R/pda.postprocess.R

```diff
@@ -0,0 +1,192 @@
+##' Postprocessing for PDA Results
+##'
+##' @title Postprocessing for PDA Results
+##' @param all params are the identically named variables in pda.mcmc / pda.emulator
+##'
+##' @return PEcAn settings list, updated with <params.id> pointing to the new params file.
+##'
+##' @author Ryan Kelly, Istem Fer
+##' @export
+pda.postprocess <- function(settings, con, mcmc.param.list, pname, prior, prior.ind) {
```

Comment:
Ultimately, it would be good to assume that functions should take in `bety` as an argument instead of `con`. Then below in the code you can use the `dplyr` functions to do queries. (Though I do see that there's also insert which would use `bety$con`)